less/layout.less

```diff
@@ -5,4 +5,48 @@ html, body {
 h1, h2, h3, h4, h5 {
     font-family: 'Russo One', sans-serif;
     color: @secondary-color;
+}
+
+.container {
+	max-width: 800px;
+}
+
+
+
+hr {
+	border: 1px solid @accent-color;
+}
+
+.btn-light {
+	background-color: #fff !important ;
+	height: 30px;
+	width: 100%;
+
+	&:hover {
```

Comment:
Love the hover effect you implemented in your project. Small details like these make a site more dimensional.